src/pages/contact-us/_offices.js

```diff
@@ -0,0 +1,251 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import { Text } from '../../components/elements/typography'
+import { Header, Image } from 'components/elements'
+import { getLocationHash } from 'common/utility'
+import { localize } from 'components/localization'
+import MapSVG from 'images/svg/map.svg'
+import PhoneSVG from 'images/svg/phone.svg'
+import { Container } from 'components/containers'
+import device from 'themes/device'
+
+const Wrapper = styled.section`
+    border-top: 1px solid #d8d8d8;
+    width: 100%;
+    height: 181.3rem;
+    background-color: var(--color-white);
+    padding: 8rem 23rem;
+
+    @media ${device.tabletL} {
+        height: auto;
+    }
+`
+const OfficesWrapper = styled(Container)`
+    display: flex;
+    flex-direction: column;
+`
+const Office = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    width: 100%;
+    height: 27.5rem;
+    margin-top: 6.3rem;
+
+    :first-child {
+        margin-top: 4rem;
+    }
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+        height: auto;
+    }
+    @media ${device.laptopL} {
+        justify-content: space-between;
+    }
+`
+const StyledHeader = styled(Header)`
+    margin-bottom: 0.8rem;
+`
+const Content = styled.div`
+    width: 50%;
+
+    :nth-child(even) {
+        margin-left: 8rem;
+    }
+    
+    @media ${device.tablet} {
+        width: 100%;
+
+        :nth-child(even) {
+            margin-left: 0;
+        }
+    }
+
+`
+const MapIcon = styled(MapSVG)`
+    margin-right: 0.8rem;
+`
+const PhoneIcon = styled(PhoneSVG)`
+    margin-right: 0.8rem;
+`
+const ImageWrapper = styled.div`
+    width: 50%;
+    height: 21.2rem;
+
+    @media ${device.tablet} {
+        display: none;
+    }
+`
+const StyledText = styled(Text)`
+    margin-bottom: ${props => props.marginBttom || '0.8rem'};
+    color: ${props => props.color || 'var(--color-black-3)'};
+    text-indent: ${props => props.textIndent || '0'};
+`
+const Splitter = styled.div`
+    background-color: #d8d8d8;
+    height: 0.1rem;
+    width: 98rem;
+`
+export const Offices = () => {
+    useEffect(() => {
```

Comment:
👍

Reply:
> @sara-fs `Email` and `Call us` section separator is not visible at first.

@arash-fs : It has another card. I'm working on it separately.